app/javascript/components/forms/components/field-wrapper/styles.scss

```diff
@@ -2,18 +2,10 @@
 
 .c-form-field {
   font-size: rem(14px);
-  margin-bottom: rem(25px);
+  margin: rem(12px) 0;
```

Comment:
Why this change?

Reply:
To add margin top to the form fields, so that they have separation on both sides (for example, to the h4 titles).

Reply:
Ok. For me the h4 titles are something custom to your individual form so should manage their layout and styles personally. Our form fields should be put components that always do the same things and make forms consistent. Also we don't have a design for the form with the new fields so the h4 titles do not make sense just yet. Please discuss with a designer.